main.py

```diff
@@ -112,11 +115,19 @@ def tick():
 ).pack(fill=tk.X)
 
 # Display Graphs:
-dates = [[]] * len(predictions.index)
-for index in range(len(predictions.index)):
-    dates[index] = predictions.index[index].strftime("%Y-%m-%d")
+###
+#dates = [[]] * len(predictions.index)
+#for index in range(len(predictions.index)):
+#    dates[index] = predictions.index[index].strftime("%Y-%m-%d")
+
+#PLACE THE SVG HERE
+cairosvg.svg2png(url='projection.svg', write_to='projection.png')
+graph_image = ImageTk.PhotoImage(Image.open("projection.png"))
+image_label = tk.Label(root, image=graph_image).pack()
+
 
-price_figure, price_ax, price_line = display_graph(root, predictions["predicted_value"], tk.LEFT, "Oil Price", "r")
+#price_figure, price_ax, price_line = display_graph(root, predictions["predicted_value"], tk.LEFT, "Oil Price", "r")
+###
```

Comment:
These commented lines above can be deleted now, although the ones below should stay as this needs to be reimplemented at some point